tests/unit/main.test.ts

```diff
@@ -16,6 +16,7 @@ describe( 'createAndMount', () => {
 			rootSelector: '#test-app',
 			licenseUrl: '',
 			licenseName: '',
+			wikibaseLexemeTermLanguages: [ 'test' ],
```

Comment:
Maybe use `en` instead of `test`? With `test` it’s not very clear that these should be language codes, IMHO.